### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Sammuel09**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,25 @@
-# myDiary
-myDairy is an online journal that allows users to make daily journal entries.
-The site is published at https://sammuel09.github.io/myDiary/
+# Project Title
+myDiary is a web-based journal application.
```
Comment (andela-ookoro) [inline README.md]: you should review the grammar here, a `web application` is more descriptive than  `web-based`.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,25 @@
-# myDiary
-myDairy is an online journal that allows users to make daily journal entries.
-The site is published at https://sammuel09.github.io/myDiary/
+# Project Title
+myDiary is a web-based journal application. 
+
+## Getting Started
+In order to have a copy of this project, clone this project to your local repository.
```
Comment (andela-ookoro) [inline README.md]: This section should assist a user to have a running copy of this project, cloning the codebase would only copy the project,  users would like to know how to set up and run the project

File: README.md [hunk 3]
```diff
@@ -1,3 +1,25 @@
...
+
+### Prerequisites
+You need to have NodeJs installed on yourlocal machine. Install Node [here](https://nodejs.org/en/)
```
Comment (andela-ookoro) [inline README.md]: I believe that your intention here is to discuss background knowledge, would a knowledge in only Nodejs be sufficient

File: README.md [hunk 4]
```diff
@@ -1,3 +1,25 @@
...
+
+### Frameworks Used
+ExpressJs. Download as an npm package [here](https://expressjs.com/)
+
+### Coding Style
+This project was built using Javascript ES6. So, to transpile the ES6 code to ES5, you need to install [BabelJS](https://babeljs.io/) an NPM package.
```
Comment (andela-ookoro) [inline README.md:16]: good job

File: README.md [hunk 5]
```diff
@@ -1,3 +1,25 @@
...
+# Project Title
+myDiary is a web-based journal application. 
+
+## Getting Started
+In order to have a copy of this project, clone this project to your local repository.
+
+### Prerequisites
+You need to have NodeJs installed on yourlocal machine. Install Node [here](https://nodejs.org/en/)
+
+### Frameworks Used
```
Comment (andela-ookoro) [inline README.md]: You should update this section when you add database and other tools. nice job

File: README.md [hunk 6]
```diff
@@ -1,3 +1,25 @@
...
+### Coding Style
+This project was built using Javascript ES6. So, to transpile the ES6 code to ES5, you need to install [BabelJS](https://babeljs.io/) an NPM package. 
+
+## Testing
+To test, you need to install [mocha](https://mochajs.org/), [chai](http://www.chaijs.com/) and [chai-http](http://www.chaijs.com/plugins/chai-http/). You also need to install [babel-register](https://www.npmjs.com/package/@babel/register) as NPM package to transpile ES6 code on the fly for testing.
+
+###
+Run the test with the NPM command: ```npm test```
+
+## Hosted
```
Comment (andela-ookoro) [inline README.md]: `Hosted` is definitely not the appropriate title for this section

**Review by andela-ookoro — CHANGES_REQUESTED** [review]
Nice job, the Readme file should have a feature section and license section. This template would guide you https://gist.github.com/PurpleBooth/109311bb0361f32d87a2

File: README.md (hunk 1 above)
Comment (Sammuel09) [reply README.md]: This is noted.

File: README.md (hunk 2 above)
Comment (Sammuel09) [reply README.md]: I have updated it now

File: README.md (hunk 3 above)
Comment (Sammuel09) [reply README.md]: I have added more information on what the user needs to know to work on the project.

File: README.md (hunk 4 above)
Comment (Sammuel09) [reply README.md:16]: Thanks

File: README.md (hunk 5 above)
Comment (Sammuel09) [reply README.md]: Yes I will.

File: README.md (hunk 6 above)
Comment (Sammuel09) [reply README.md]: OK. Sir.

**Review by andela-ookoro — CHANGES_REQUESTED** [review]
The branch name is wrong, this is the naming convention `{story type}-{story summary}-{pivotal tracker id} e.g ft-resources-rest-endpoints-111504508`

**Pull request closed without merging**